app/src/main/java/cm/aptoide/pt/editorialList/CustomLinearLayoutManager.java

```diff
@@ -0,0 +1,20 @@
+package cm.aptoide.pt.editorialList;
+
+import android.content.Context;
+import android.support.v7.widget.LinearLayoutManager;
+
+public class CustomLinearLayoutManager extends LinearLayoutManager {
```

Comment:
I dislike this name as it may be useful to reuse this in the future and the name should make clear what the purpose is